Eyes/src/Modules/EyesWrapper.cs

```diff
@@ -86,27 +86,27 @@ public static void CheckFolder(string fileOrFolderPath)
             suite.Run(eyes);
         }
 
-        public static void CloseTest(bool throwException)
+        public static void CloseTest()

```

Comment:
Is this a breaking change in the public interface? If so, we need to adjust the version number correctly

Reply:
Right, sure. I don't know if it's actually used in such fashion, normal workflow is that it is called from FinalizeEyes in TearDown and StartOrContinueTest which are also provided with the package. Any preference on the version number, e.g. 1.1.0 or 1.0.1?

Reply:
If it's really a breaking change (and I'm not familiar enough with the code to decide that), than use 2.0.0.
Otherwise use default SemVer:
if feature added since last version number, increment 2nd number.
if only bug fixes since last veresion number, increment 3rd number.

Reply:
IMHO this is more of feature add, because the validation didn't do anything. It's only a breaking change as far as the call to the method, but the actual behavior of the method didn't change. So I don't see it as a breaking change.

So I would just leave it as it is now with the Obsolete attribute and bump the minor version.

Reply:
OK so from start of this discussion thread, here's the summary of changes: keep the old method, decorated it with Obsolete (warning), and increased the version to 1.1.0
Could you please advise if you agree with this approach and if this can be considered resolved?
Thanks